Library/Homebrew/cmd/outdated.rb

```diff
@@ -20,46 +22,79 @@ def outdated_args
       switch :verbose,
              description: "Include detailed version information."
       flag   "--json",
-             description: "Print output in JSON format. Currently the default and only accepted "\
-                          "value for <version> is `v1`. See the docs for examples of using the JSON "\
-                          "output: <https://docs.brew.sh/Querying-Brew>"
+             description: "Print output in JSON format. There are two versions: v1 and v2. " \
+                          "v1 is deprecated and is currently the default if no version is specified. " \
+                          "v2 prints outdated formulae and casks. "
```

Comment:
```suggestion
                          "v1 prints outdated formulae. " \
                          "v2 prints formulae and casks and is currently the default if no version is specified. "
```

Reply:
Just to confirm: should v2 be made the default? v1 is currently the default for compatibility's sake

Reply:
Yeh, I think so. We'll give it a bit of time before the next time to see if anything breaks but I don't believe this JSON output is widely used. Might be a bit of a waste of your time but we could even consider fully deprecating the JSON output and see if anyone complains?

Reply:
I'm totally fine with deprecating the json flag as a whole even with the the time I put in. Do you think making v2 the default should happen now or later? In my opinion v1 should stick a round for a moment just to be conservative.

Reply:
Let me do a quick poll of maintainers on Slack.

Reply:
Yeh, this is used unfortunately. As a result here's my proposed plan:

- make v1 the default for now but mark it deprecated in the docs and add a (commented out) `odeprecated` message which we'll enable at the next major/minor release.